resources/js/app.js

```diff
@@ -102,4 +102,42 @@ $(document).ready(function () {
             $("#address_input").focus();
         }
     });
+
+    //Read more
+    // Configure/customize these variables.
+    var showChar = 500;  // How many characters are shown by default
+    var ellipsesText = "...";
+    var moreText = "Xem thêm";
+    var lessText = "Ẩn bớt";
+    
+
+    $('.more').each(function () {
+        var content = $(this).html();
+ 
+        if (content.length > showChar) {
+ 
+            var beforeText = content.substr(0, showChar);
+            var afterText = content.substr(showChar, content.length - showChar);
+ 
+            var html = beforeText + '<span class="moreellipses">' + ellipsesText+ '&nbsp;</span><span class="morecontent"><span>' + afterText + '<br></span><button class="btn btn-info morelink">' + moreText + '</button></span>';
+ 
```

Comment:
```suggestion
            let beforeText = content.substr(0, showChar);
            let afterText = content.substr(showChar, content.length - showChar);
 
            let html = beforeText + '<span class="moreellipses">' + ellipsesText + '&nbsp;</span><span class="morecontent"><span>' + afterText + '<br></span><button class="btn btn-info morelink">' + moreText + '</button></span>';
 
```
Em comment chút và có vài suggestion cho xử lý chuỗi dài:

**Suggestion:** em khuyến nghị để đơn giản hóa toàn bộ các bước này anh có thể sử dụng text-truncate của Bootstrap: https://getbootstrap.com/docs/4.3/utilities/text/#text-wrapping-and-overflow

Reply:
ready